src/sas/sascalc/dataloader/loader.py

```diff
@@ -72,8 +73,19 @@ def load(self, path, format=None):
                 ascii_loader = ascii_reader.Reader()
                 return ascii_loader.read(path)
             except:
-                cansas_loader = cansas_reader.Reader()
-                return cansas_loader.read(path)
+                try:
+                    cansas_loader = cansas_reader.Reader()
+                    return cansas_loader.read(path)
+                except:
```

Comment:
Shouldn't use bare "except:" since it will trap SystemExit and KeyboardInterrupt.  At a minimum use "except Exception:", but more specific is better if you know the specific failure possibilities for the called code.

Also, this looks like a case where "pass" is an acceptable exit from the exception.  This will keep down the indent level.  Example:
```
    try:
        load format 1
    except Exception:
        pass  # fall through to next format
    try:
        load format 2
    except Exception:
        pass  # fall through to next format
    ...
```